### PR TITLE
Added bot discoverability

### PR DESCRIPTION
**Pull request by leotrj**
You can now discover bot(s?)

### CODE REVIEW

File: gui/index2.html [hunk 1]
```diff
@@ -35,8 +35,8 @@
 		<block type="wait"></block>
 	</xml>
 	<!-- END HTML -->
-	<script src="./static/js/interface.js"></script>
-	<script src="./static/js/view.js"></script>
+	<!--<script src="./static/js/interface.js"></script>-->
```
Comment (leotrj) [inline gui/index2.html]: These are interfering with functions in React

File: gui/main.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def make_app(self):
 class BaseStationHandler(tornado.web.RequestHandler):
     def get(self):
         # self.write("Hi There")
-        self.render("../gui/index.html", title="Title", items=[])
+        self.render("../gui/index2.html", title="Title", items=[])
```
Comment (leotrj) [inline gui/main.py]: Index2 refers to the new html and index refers to our old one

File: gui/index2.html (hunk 1 above)
Comment (chynu) [reply gui/index2.html]: Can delete if unnecessary

File: gui/main.py (hunk 2 above)
Comment (chynu) [reply gui/main.py]: Please rename old index to `index_old.html` and this one to `index.html`.

File: gui/static/js/components/addbot.js [hunk 3]
```diff
@@ -42,6 +44,10 @@ export default class AddBot extends React.Component {
         this.updateDiscoveredBots = this.updateDiscoveredBots.bind(this);
     }
 
+    componentWillMount() {
```
Comment (chynu) [inline gui/static/js/components/addbot.js]: Please add jsdoc.

**Review by chynu — APPROVED** [review]
lgtm

**Pull request merged**